src/pages/profile/external-identities/FrameworkServiceLink.ts

```diff
@@ -0,0 +1,47 @@
+import { IconProps } from '@terra-ui-packages/components';
+import { Fragment, useState } from 'react';
+import { h } from 'react-hyperscript-helpers';
+import { ButtonPrimary, Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import { Ajax } from 'src/libs/ajax';
+import { withErrorReporting } from 'src/libs/error';
+import { useOnMount } from 'src/libs/react-utils';
+import * as Utils from 'src/libs/utils';
+
+interface FrameworkServiceLinkInputs extends IconProps {
```

Comment:
Conventionally, types for component props are named with `Props`.

```suggestion
interface FrameworkServiceLinkProps extends IconProps {
```

Also, it looks like the props are being passed through to either ButtonPrimary or Link, so this should probably extend `ClickableProps` instead of `IconProps`.